data/xml/E03.xml

```diff
@@ -3,6 +3,12 @@
   <volume id="1">
     <meta>
       <booktitle>10th Conference of the <fixed-case>E</fixed-case>uropean Chapter of the Association for Computational Linguistics</booktitle>
+      <editor><first>Ann</first><last>Copestake</last></editor>
+      <editor><first>Jan</first><last>Hajic</last></editor>
```

Comment:
Should that be Hajič instead of Hajic?

Reply:
Hmm, I copied it from [this page](http://www.conferences.hu/EACL03/start.htm), but now that you mention it, yes.